src/components/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type Props = {
+  active?: boolean;
+  styleClass: string;
+  onClick: () => void;
+  children: string;
+};
+
+const Button: React.FC<Props> = ({ active, styleClass, onClick, children }) => {
+  return (
+    <button
+      type="button"
+      className={`button ${styleClass} ${active ? '' : 'is-light'}`}
```

Comment:
It's better to use classnames library to set classes conditionally https://www.npmjs.com/package/classnames